daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -559,6 +559,17 @@
             </xs:restriction>
           </xs:simpleType>
         </xs:element>
+        <xs:element name="xmlOutputStyle" type="xs:string" default="none" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>
+              values is a whitespace separated list of tokens drawn from this set.
+              Values are:
+              - none: (Current behavior - ok if data is not being pretty printed, or will not be re-read in, or if whitespace is fungible in the actual data format),
```

Comment:
Actually, I think "none" should be "normal" or "standard".

Reply:
The file dafext.xsd seems no longer to be in the change set.

I'm not sure what's going on here, but that file *should* be in this change set so that the above comment can be updated.

Reply:
Tunables aren't used to change this settings anymore, it's now part of the XMLTextInfosetOutputter constructor.